test/controllers/DoYouWantToRemoveCountryControllerSpec.scala

```diff
@@ -35,12 +36,12 @@ import scala.concurrent.Future
 
 class DoYouWantToRemoveCountryControllerSpec extends SpecBase with MockitoSugar {
   val taxYear = 2024
-  def onwardRoute = Call("GET", "/foo")
-  val country = Country("Spain", "ESP")
+  def onwardRoute: Call = Call("GET", "/update-and-submit-income-tax-return/property")
```

Comment:
The `onwardRoute` could remain as `Call("GET", "/foo")` once the Navigator is overriden with `ForeignPropertyNavigator` and `FakeForeignPropertyNavigator`